src/Main.elm

```diff
@@ -154,7 +162,14 @@ viewCard title shadow { open, selected } menu =
 
 view : Model -> Html Msg
 view model =
-    div []
+    div
+        (case model.open of
+            Nothing ->
+                []
+
+            Just _ ->
+                [ onClick OnClickOutside ]
+        )
```

Comment:
```suggestion
    div (Maybe.withDefault [] <| Maybe.map (onClick << OnClickOutside) model.open)
```

Reply:
I was wondering whether the `maybe` fn existed or not, thanks! 👏🏻